INDLinkLabel.swift

```diff
@@ -141,17 +149,24 @@ import UIKit
             if let URL = URL {
                 let glyphRange = self.layoutManager.glyphRangeForCharacterRange(range, actualCharacterRange: nil)
                 ranges.append(LinkRange(URL: URL, glyphRange: glyphRange))
-                
+
                 // Remove `NSLinkAttributeName` to prevent `UILabel` from applying
                 // the default styling.
-                let attributes = self.textStorage.attributesAtIndex(range.location, effectiveRange: nil)
-                if attributes[NSForegroundColorAttributeName] == nil {
-                    self.textStorage.addAttribute(NSForegroundColorAttributeName, value: DefaultLinkAttributes.Color, range: range)
+                self.textStorage.removeAttribute(NSLinkAttributeName, range: range)
```

Comment:
Style: omit `self.` prefix when referring to a property (same for the other uses below)